tests/README.md

```diff
@@ -179,9 +179,15 @@ internal_bench/bytebuf:
 
 SSL/TLS tests in `multi_net` and `net_inet` use a
 self-signed key/cert pair that is randomly generated and to be used for
-testing/demonstration only. You should always generate your own key/cert.
+testing/demonstration only.
 
-To generate a new self-signed RSA key/cert pair with openssl do:
+To use these on-device the der files should be copied and the current time
+set to ensure certs validity. This can be done with:
+```
+mpremote a0 rtc --set cp multi_net/*.der :
```

Comment:
I think it should also copy the `net_inet` files.  Also probably drop the `a0` because that could be confusing.  If users don't know how to use mpremote they need to read its documentation.
```
mpremote rtc --set cp multi_net/*.der net_inet/*.der :
```
The above should still work with port auto-detection.

Reply:
Thanks I've updated as such and clarified the need / usage of these certs a bit further.

Reply:
Great, thanks for updating!